spec/substantive-changes-summary.txt

```diff
@@ -0,0 +1,20 @@
+**********************************
+*                                *
+* Summary of substantive changes *
```

Comment:
This should be included in the main text, like [other specs](https://www.w3.org/TR/css-values-3/#changes).

Reply:
Why? We have adopted this approach before in TTWG, for example at https://www.w3.org/TR/ttml2/ttml2-changes.html linked from TTML2 Rec. Is there some recommendation or guide that explains this more?

Reply:
Actually, not so string in my mind, but 1. aligning with other publications makes reader easier, 2. I doubt streamlined publication tool will not take this type of external asset well...

Reply:
@nigelmegitt @himorin Please pick one approach. The editor does not really care but does not want to have to do it twice.

Reply:
I don't care either. Apologies for the unnecessary challenge @himorin , I'm happy to incorporate the changes into the document itself.

Reply:
Thank you for changing on this.
I'd suggest to have some text to say about the period for this list, like FPWD to CR1? (we may publish more, incl. PR, with some changes, and we need to have dedicated list for these publication)

Reply:
I do not think it is useful for the reader (outside of W3C AC reps) to have a list of changes from FPWD to CR1 to PR to REC, which is why it was nice to have that information in a separate file.

Reply:
When I check some other specifications for updates for a while (e.g. getting updated for recent development), lists in changes section are somehow useful. Also to perform horizontal review (and should be the same for wide reviews), it is quite useful and highly appreciated if spec has such list, especially for delta review.
These would not be useful for ones who just read the completed and published REC, of course, although...

Reply:
Ok. Will do like https://www.w3.org/TR/css-values-3/#changes.

Reply:
@himorin See https://github.com/w3c/imsc-hrm/pull/59/commits/b5e6d84d3886323d683a63b8a8fd33da8be73aa0

Reply:
Thank you so---o much! m__m